components/ActiveLocalizedLink.tsx

```diff
@@ -0,0 +1,61 @@
+import { useRouter } from 'next/router';
```

Comment:
You should put this file on:
```
activeLocalizedLinke/index.tsx
```

and unit test it

Reply:
just as is, or should it go in one of the other folders?